ifrs17/Utils/Extensions.ipynb

```diff
@@ -96,7 +123,16 @@
         {
             "cell_type": "code",
             "source": [
-                "public static double[] Prune (this IEnumerable<double> source, double precision = Precision) => source.Reverse().SkipWhile(x => Math.Abs(x) < precision).Reverse().ToArray();"
+                "public static double[] Prune(this IEnumerable<double> source, double precision = Precision) => source.Reverse().SkipWhile(x => Math.Abs(x) < precision).Reverse().ToArray();"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static double[] AggregateDoubleArray(this IEnumerable<IEnumerable<double>> source) => source.Aggregate((x, y) => x.ZipLongest(y, (a, b) => a + b)).ToArray();"
```

Comment:
@sbusato Here is the extension --> We can add very simple tests, I might even have them already